components/usage-api/usage/v1/billing.proto

```diff
@@ -50,3 +53,17 @@ message FinalizeInvoiceRequest {
 
 message FinalizeInvoiceResponse {
 }
+
+// If there are two billable sessions for this instance ID,
+// the second one's "from" will be the first one's "to"
+message SetBilledSessionRequest {
+  string instance_id = 1;
+  google.protobuf.Timestamp from = 2;
+  oneof system {
```

Comment:
@easyCZ Is this correct?

Reply:
And the value of either `chargebee` or `stripe` would be the Invoice ID, right?

Reply:
If we do not care about the value then using an Enum may be simpler but I think we'll still want to capture the invoice ID

Reply:
> And the value of either `chargebee` or `stripe` would be the Invoice ID, right?

It would be the string "chargebee" or "stripe"

Reply:
> we'll still want to capture the invoice ID

The invoice ID won't be relevant for "chargebee" though because we will be excluding them anyway, right?

Reply:
> It would be the string "chargebee" or "stripe"

Ah, that's not how the `oneof` works. It basically allows you to set at most one of the fields in it's block. But you're still setting them to a value.

I think use an Enum here for now, 
```
enum System {
    UNKNOWN = 0;
    CHARGEBEE = 1;
    STRIPE = 2;
 }
  System system = 4;
```

The reason for making Unknown 0 value is because the 0th entry will also be the default value and here we don't want to default to either of chargebee or stripe (we want it to be explicitly set by the caller)